utils.py

```diff
@@ -1,30 +1,27 @@
 import json
 import logging
-import os
 from urllib.request import urlopen
 
-from dotenv import load_dotenv
+from airflow.models import Variable
 from minio import Minio
 from operators.elastic_create_siren import ElasticCreateSirenOperator
 from operators.elastic_fill_siren import ElasticFillSirenOperator
 from operators.papermill_minio_siren import PapermillMinioSirenOperator
 
-load_dotenv()
-
-AIRFLOW_URL = os.getenv("AIRFLOW_URL")
-COLOR_URL = os.getenv("COLOR_URL")
-ELASTIC_PASSWORD = os.getenv("ELASTIC_PASSWORD")
-ELASTIC_URL = os.getenv("ELASTIC_URL")
-ELASTIC_USER = os.getenv("ELASTIC_USER")
-MINIO_BUCKET = os.getenv("MINIO_BUCKET")
-MINIO_PASSWORD = os.getenv("MINIO_PASSWORD")
-MINIO_URL = os.getenv("MINIO_URL")
-MINIO_USER = os.getenv("MINIO_USER")
-DAG_FOLDER = os.getenv("DAG_FOLDER")
-DAG_NAME = os.getenv("DAG_NAME")
-AIRFLOW_DAG_HOME = os.getenv("AIRFLOW_DAG_HOME")
-TMP_FOLDER = os.getenv("TMP_FOLDER")
-ENV = os.getenv("ENV")
+AIRFLOW_URL = Variable.get("airflow_url")
+COLOR_URL = Variable.get("color_url")
+ELASTIC_PASSWORD = Variable.get("elastic_password")
+ELASTIC_URL = Variable.get("elastic_url")
+ELASTIC_USER = Variable.get("elastic_user")
+MINIO_BUCKET = Variable.get("minio_bucket")
+MINIO_PASSWORD = Variable.get("minio_password")
+MINIO_URL = Variable.get("minio_url")
+MINIO_USER = Variable.get("minio_user")
+DAG_FOLDER = Variable.get("dag_folder")
+DAG_NAME = Variable.get("dag_name")
+AIRFLOW_DAG_HOME = Variable.get("airflow_dag_home")
+TMP_FOLDER = Variable.get("tmp_folder")
+ENV = Variable.get("env")
```

Comment:
all in uppercase please. And no need for tmp_folder, airflow_dag_home, dag_name, dag_folder to be variables (could be in clear I suppose)